File: keychain-sample-chat-dot-net/KeychainChat/MVVM/Model/Personas/PendingPersona.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Keychain;

namespace KeychainChat.MVVM.Model.Personas
{
    public class PendingPersona
    {
        public Persona? persona;
        public IEnumerable<byte>? image;        // For future use
        public string recordId;

        public PendingPersona(string id, Persona? persona, IEnumerable<byte>? image)
        {
            recordId = id;
            this.persona = persona;
            this.image = image;
        }

    }
}

File: keychain-sample-chat-android/app/src/main/java/io/keychain/chat/models/PendingPersona.java
package io.keychain.chat.models;

import android.graphics.Bitmap;

import io.keychain.core.Persona;

public class PendingPersona {
    public Persona persona;
    public Bitmap image;        // For future use
    public String recordId;

    public PendingPersona(String id, Persona persona, Bitmap image) {
        this.recordId = id;
        this.persona = persona;
        this.image = image;
    }
}
